src/services/errorTemplate.ts

```diff
@@ -37,6 +45,8 @@ export default class ErrorTemplateService {
 
             await this.store.bumpTags([StoreTag.ErrorTemplate])
 
+            this.errorsCount += 1
```

Comment:
It won't work in a high-availability setup when multiple instances run concurrently. I don't see valuable reasons in attempts to optimize count query on the collection with tens of records